### DISCUSSION TITLE
页眉 与正文 重合的重合问题

### DISCUSSION THREAD

**Discussion by moSODOPE** [tuna/thuthesis#733] [Q&A] [answered]
当我的章标题比较长时，页眉会分行，就导致页眉与正文重合，如下图
<img width="618" alt="image" src="https://user-images.githubusercontent.com/50304484/156350231-21d8f11e-81b4-4b1a-9acb-4fdee045f7f9.png">
学校对页眉的要求是：
<img width="446" alt="image" src="https://user-images.githubusercontent.com/50304484/156350896-24a43511-a503-49de-bb12-49a054144419.png">
是不是要使得多行页眉时，页眉内容自动向上顶，而不是像第一张图那样向下顶导致与正文重合了？这种设置要如何操作呢，或者怎么解决这个重合问题呢，非常感谢！

**Comment by zepinglee** [comment, marked as answer]
一般不会将页眉向上移。建议优先考虑将标题改写为一行。其次，可以用 `\chapter[Short Heading]{Long Heading}` 在页眉和目录使用较短的标题，但是可能与《指南》的要求不符。

**Reply by moSODOPE** [reply]
word里的模板遇这种情况是 页眉的横线不会动，只是页眉的文字内容逐行上移，这样是不符合latex逻辑或者《指南》吗，  无论如何，非常感谢！
<img width="865" alt="image" src="https://user-images.githubusercontent.com/50304484/156482111-0ca7aa95-0d9e-49a5-9105-b940a1d946fc.png">

**Reply by zepinglee** [reply]
目前模板的页眉是沿底部对齐的：
```latex
\documentclass[degree=master,language=english]{thuthesis}
\begin{document}
\chapter{A Long long long long long long long long long long long long long
  long long long long long long long long long long long long long long long
  long long long long long long long title
}
\end{document}
```
<img width="664" alt="Screen Shot 2022-03-03 at 12 31 39" src="https://user-images.githubusercontent.com/12290822/156496578-1fea62a7-bf85-438d-a586-9f3955d3da25.png">

请确认一下你的模板版本和编译环境。
